pytorch_transformers/modeling_utils.py

```diff
@@ -336,9 +340,17 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
                 configuration should be cached if the standard cache should not be used.
             **output_loading_info**: (`optional`) boolean:
                 Set to ``True`` to also return a dictionnary containing missing keys, unexpected keys and error messages.
+            **model_args**: (`optional`) Sequence:
+                All positional arguments will be passed to the underlying model's __init__ function
             **kwargs**: (`optional`) dict:
-                Dictionnary of key, values to update the configuration object after loading.
-                Can be used to override selected configuration parameters. E.g. ``output_attention=True``
+                Dictionary of key, values to update the configuration object after loading.
+                Can be used to override selected configuration parameters. E.g. ``output_attention=True``.
+
+                If config is None, then **kwargs will be passed to the model.
```

Comment:
Not sure that `if config is None, then **kwargs will be passed to the model`.
Maybe rather 
```
- If a configuration is provided with `config`, **kwargs will be directly passed
to the underlying model's __init__ method.
- If a configuration is not provided, **kwargs will be first passed to the pretrained
model configuration class loading function (`PretrainedConfig.from_pretrained()`).
Each key of **kwargs that corresponds to a configuration attribute
will be used to override said attribute with the supplied **kwargs value.
Remaining keys that do not correspond to any configuration attribute will
be passed to the underlying model's __init__ function.
```

Reply:
Changed in https://github.com/huggingface/pytorch-transformers/pull/866#issuecomment-514245282